### PR TITLE
Fix ES6 to ES4 conversion

### PR DESCRIPTION
**Pull request by alisonhall**
Fixes issue #136 

Instead of naively using a bash script to remove all lines that start with `import` or `export`, I am using the `jscodeshift` package. This allows us to remove multi-line import/export statements, as well as removing only the `export` keyword that might come before a function declaration.

Note: Running this updated code has made some minor changes to the existing ES4 files; sometimes adding some additional brackets around modules, and sometimes remove some empty lines; no functionality should be impacted. It does remove the old ES4 hamburger.js file, since there is no matching ES6 file anymore.

Additional Note: there are 2 existing files (enable-carousel.js and enable-dialog.js) where there are import statements embedded within the module code, with `then` and `catch` statements coming off of the import. These import statements weren't being removed with the original bash script, and they're not being removed using this new script either. The modules need to be refactored in order for the import statements to be detached from the rest of the modules code.

### CODE REVIEW

File: bin/convert-modules-to-libs.sh [hunk 1]
```diff
@@ -2,23 +2,11 @@
...
-for i in $FILES
-do
-  echo "Converting $i"
-  grep -v ^import $i | grep -v ^export > es4/$i
-done
+# Duplicate the ES6 files into the ES4 folder
+cp js/modules/*.js js/modules/es4/
 
+# Convert the ES6 files to ES4 using jscodeshift
+npx jscodeshift -t bin/remove-imports-exports.js js/modules/es4/*.js
```
Comment (zoltan-dulac) [inline bin/convert-modules-to-libs.sh:12]: Nice.  Didn't know this was a thing.

File: js/modules/es4/paginate.js [hunk 2]
```diff
@@ -1,20 +1,6 @@
...
- * More information about this script available at:
- * https://www.useragentman.com/enable/table.php
- * 
- * Released under the MIT License.
- ******************************************************************************/
-
-
-const paginationTables = new function() {
+const paginationTables = new (function() {
   let perPage = 20;
```
Comment (zoltan-dulac) [inline js/modules/es4/paginate.js:4]: Is there a way to keep these comments in?  They contain license information.

**Review by zoltan-dulac — COMMENTED** [review]
This is really good ... just one question about the multiline /* */ comments.

File: js/modules/es4/paginate.js (hunk 2 above)
Comment (alisonhall) [reply js/modules/es4/paginate.js:17]: Sure! I've now updated the script to keep those comments 😄

File: bin/convert-modules-to-libs.sh (hunk 1 above)
Comment (alisonhall) [reply bin/convert-modules-to-libs.sh:12]: I didn't either! GitHub Copilot to the rescue!

**Review by zoltan-dulac — APPROVED** [review]

**Pull request merged**